routes/tools_routes.js

```diff
@@ -257,12 +258,22 @@ router.get('/card/:id', async (req, res) => {
     const pids = carddb.nameToId[card.name_lower].map((id) => carddb.cardFromId(id).tcgplayer_id);
     const prices = await GetPrices(pids);
     card.elo = (await getElo([card.name], true))[card.name];
-    return res.render('tool/cardpage', {
+    const reactProps = {
       card,
       data,
       prices,
       cubes,
       related: data.cubedWith.map((name) => carddb.getMostReasonable(name[0])),
+    };
+    return res.render('tool/cardpage', {
+      reactProps: serialize(reactProps),
+      title: `${card.name}`,
+      metadata: generateMeta(
+        `${card.name} - Cube Cobra`,
+        `Analytics for ${card.name} on CubeCobra`,
+        card.art_crop,
```

Comment:
I think I like using the full card art here

Reply:
Done!  I flipped back and forth a few times.  Let's see how it looks on the dev server -- i wasn't sure if the dimensions would be right.